### PR TITLE
Language Generation - C#, JS samples

### PR DESCRIPTION
**Pull request by vishwacsena**
- [x] Updated all samples to 4.9.0
- [x] Moved samples to new location.

| Current location                                                                                   | Proposed location                                                                                                                                            |
|----------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
| C# samples: experimental/language-generation/csharp_dotnetcore/                                    | C# samples: samples/csharp_dotnetcore/language-generation                                                                                                    |
| JS samples: experimental/language-generation/javascript_nodejs/                                    | JS samples: samples/javascript_nodejs/language-generation                                                                                                    |
| Documentation: experimental/language-generation/README.md & experimental/language-generation/docs/ | Documentation: Official bot builder docs. Add a README.md under experimental/language-generation/ to point back to stable release docs and samples location. |

### CODE REVIEW

File: experimental/language-generation/README.md [hunk 1]
```diff
@@ -1,224 +1,15 @@
-# Language Generation ***_[PREVIEW]_***
+# Language Generation (**Stable release**)
 
-> See [here](#Change-Log) for what's new in **4.8.0 RC1** release.
+This feature is no longer experimental. It is available in **Stable release**, starting botbuilder SDK version 4.9.0 and higher.
```
Comment (tomlm) [inline experimental/language-generation/README.md:3]: >This feature is no longer experimental [](start = 0, length = 38)

I would change the verbiage a bit around this.  Elevator statement of what language generation is with link to docs.

File: samples/csharp_dotnetcore/language-generation/05.a.multi-turn-prompt-with-language-fallback/AdapterWithErrorHandler.cs [hunk 2]
```diff
@@ -8,12 +8,13 @@
...
 using Microsoft.Bot.Builder.Integration.AspNet.Core;
 using System.Collections.Generic;
+using Microsoft.Bot.Builder.LanguageGeneration;
 
 namespace Microsoft.BotBuilderSamples
 {
     public class AdapterWithErrorHandler : BotFrameworkHttpAdapter
     {
-        private MultiLingualTemplateEngine _lgManager;
+        private MultiLanguageLG _lgManager;
```
Comment (tomlm) [inline samples/csharp_dotnetcore/language-generation/05.a.multi-turn-prompt-with-language-fallback/AdapterWithErrorHandler.cs:17]: >MultiLanguageLG [](start = 16, length = 15)

what the Wha?  How did this name ship?  We don't use acronyms in our class names!

Comment (vishwacsena) [reply samples/csharp_dotnetcore/language-generation/05.a.multi-turn-prompt-with-language-fallback/AdapterWithErrorHandler.cs:17]: :( we could deprecate this in the future and fix up. Will file a tracking ticket.

File: experimental/language-generation/README.md (hunk 1 above)
Comment (vishwacsena) [reply experimental/language-generation/README.md:3]: Updated.

**Pull request merged**